tests/scripts/thread-cert/thread_cert.py

```diff
@@ -522,10 +524,36 @@ def _has_backbone_traffic(self):
         return False
 
     def _prepare_backbone_network(self):
-        network_name = config.BACKBONE_DOCKER_NETWORK_NAME
-        self.assure_run_ok(
-            f'docker network create --driver bridge --ipv6 --subnet {config.BACKBONE_PREFIX} -o "com.docker.network.bridge.name"="{network_name}" {network_name} || true',
-            shell=True)
+        """
+        Prepares one or multiple backbone network(s).
+
+        This method creates the backbone network based on the `backbone` values defined in the TOPOLOGY in each test.
+        If no backbone values are defined, it sets the default backbone network as BACKBONE_DOCKER_NETWORK_NAME
+        from the config module.
+
+        Args:
+            None
+
+        Returns:
+            None
+        """
+        # Use backbone_set to store all the backbone values in TOPOLOGY
+        backbone_set = set()
+        for node in self.TOPOLOGY:
+            backbone = self.TOPOLOGY[node].get('backbone_network')
+            if backbone:
+                backbone_set.add(backbone)
+
+        # Set default backbone network if backbone_set is empty
+        if not backbone_set:
+            backbone_set.add(config.BACKBONE_DOCKER_NETWORK_NAME)
+
+        # Iterate over the backbone_set and create the backbone network
+        for offset, backbone in enumerate(backbone_set, start=PORT_OFFSET):
+            backbone_prefix = f'{config.BACKBONE_IPV6_ADDR_START_BASE + offset:04x}::/64'
```

Comment:
`PORT_OFFSET` is not used here. Is it as intended?

Reply:
`offset` actually will initially assigned as `PORT_OFFSET`